packages/design/src/Typography.stories.mdx

```diff
@@ -101,11 +100,11 @@ labels, and more!
   </Content>
 </Canvas>
 
-## Font sizes
+## Font-size (web)
```

Comment:
Thoughts on dropping this web and mobile thing on a tab? 
![image](https://github.com/GetJobber/atlantis/assets/15986172/f3872cc5-5ec7-47c2-8e19-115bf08f0b64)

Reply:
I like it!